Packs/DomainTools_Iris/Integrations/DomainTools_Iris/DomainTools_Iris.py

```diff
@@ -89,36 +98,14 @@ def get_dbot_score(proximity_score, age, threat_profile_score):
     else:
         return 1
 
+def should_include_context(include_context):
+    if include_context is False:
+        return False
```

Comment:
We've discussed this and concluded that this is problematic, since reputation commands run automatically by the server to enrich indicators, and adding this flag will make it behave unexpectedly.

Please update it to return indicators in any case.
As for the performance issues you had, which are why you added this, please reach out to me on Slack, and we'll see what the issue is and how we can resolve it.

Reply:
We've updated the `should_include_context` to default to `True`. We thought this would allow automatic runs to work as expected, while giving users the option to turn it off for a single manual run if they run into performance issues. What do you think?